Fix #12895: Prevent Mechanics from fixing just fixed rides (#13389)
OpenRCT2/OpenRCT2: distribution/changelog.txt

- Feature: [#13057] Make GameAction flags accessible by plugins.
- Feature: [#13376] Open custom window at specified tab.
- Change: [#13346] Change FootpathScenery to FootpathAddition in all occurrences.

> - Fix: [#12895] Mechanics no longer are called to repair rides that were just re-opened

Reviewer: ```suggestion
- Fix: [#12895] Mechanics are called to repair rides that have already been fixed.
```
We always describe the bug here, not the solution.
